bitcoin/examples/create-p2wpkh-address.rs

```diff
@@ -0,0 +1,27 @@
+extern crate bitcoin;
+
+use bitcoin::{KnownHrp, Address, CompressedPublicKey, PrivateKey, Network};
+use bitcoin::secp256k1::{rand, Secp256k1};
+
+// Generate a P2WPKH (pay-to-witness-public-key-hash) address and print it
+// along with the associated private key needed to transact.
+
+fn main() {
+    // Create a new secp256k1 elliptic curve.
+    let s = Secp256k1::new();
+
+    // Generate a secp256k1 secret key.
+    let (secret_key, public_key) = s.generate_keypair(&mut rand::thread_rng());
+
+    // Create a Bitcoin private key.
+    let private_key = PrivateKey::new(secret_key, Network::Bitcoin);
+
+    // Create a compressed Bitcoin public key derived from the secp256k1 public key.
+    let public_key = CompressedPublicKey(public_key);
+
+    // Create a Bitcoin P2WPKH address.
+    let address = Address::p2wpkh(public_key, KnownHrp::Mainnet);
```

Comment:
Most people won't know what is KnownHrp. Network is more example-friendly than Hrp stuff 
```rust
let network = Network::Bitcoin;
let private_key = PrivateKey::new(secret_key, network);
...
let address = Address::p2wpkh(public_key, KnownHrp::Mainnet);
```